nunit.sln

```diff
@@ -258,6 +272,10 @@ Global
 		{08D0F53A-CD37-4060-B5A2-BCC202F0DE46}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{08D0F53A-CD37-4060-B5A2-BCC202F0DE46}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{08D0F53A-CD37-4060-B5A2-BCC202F0DE46}.Release|Any CPU.Build.0 = Release|Any CPU
+		{DBD0C8B0-BE4E-4CBF-AAD4-44BEA2EA830B}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
All of the changes in the solution just look like MonoDevelop re-arranging things. You might want to revert all of your changes to this file otherwise merging in latest from master is going to be impossible.